main.go

```diff
@@ -52,6 +53,24 @@ func (c config) AssertValid(pattern string) {
 	}
 }
 
+var (
+	// Statsd is a globally-shared datadog client.
+	Statsd *dogstatsd.Client
+)
+
+func configureStatsd() {
+	var err error
+	if Statsd != nil {
+		return
+	}
+	Statsd, err = dogstatsd.New("127.0.0.1:8125", &dogstatsd.Context{
+		Namespace: "sv-rollout.",
+	})
+	if err != nil {
+		log.Fatal(err)
```

Comment:
I think it's okay that it aborts. It takes a pretty serious error to not be able to "connect" to a UDP socket ;)